superflore/utils.py

```diff
@@ -155,6 +155,8 @@ def get_license(l):
     boost_re = '^(Boost)((.)*([1]))?'
     pub_dom_re = '^(Public(.)?Domain)'
     mit_re = '^MIT'
+    government_re = '.*Government.*'
```

Comment:
This is a little broad for capturing GPR in general. I also don't really consider GPR a license. As it is not something that applies to anyone else besides the US Government.  And it's specific to the deliverable provided by a contractor. It doesn't have carry on effects for future licensees or changes.